### ISSUE TITLE
Make the Yunohost Tile available

### ISSUE THREAD

**Issue by yalh76** [YunoHost-Apps/pixelfed_ynh#41]

**Comment by lapineige** [comment]
What's needed to achieve that ?

**Comment by yalh76** [comment]
> 
> 
> What's needed to achieve that ?

As far I've seen, request to the tile is made to pixelfed instead of downloading them ...

**Comment by lapineige** [comment]
Uh what ? 😅

**Comment by yalh76** [comment]
when you look at pixefed log doing `journactl -feu pixelfed` you will see the request to the files: `custom_portal.js`, `ynh_portal.js`, `custom_portal.css`, `ynh_portal.css` but pixelfed don't know where thoses files are.... 

So nginx configuration has to be changed, for those files not be requested to pixelfed but directly  provided ...

I'm not sure, I'me more clear ^^

**Comment by xplosionmind** [comment]
This issue is still relevant for YunoHost 12. There appears to be no tile for Pixelfed, even when it is set to be displayed in the user portal!

**Comment by lapineige** [comment]
I believe the next Yunohost version will remove it :thinking:

**Comment by xplosionmind** [comment]
> I believe the next Yunohost major version (for debian 12) will remove it 🤔

I don’t think so. I just updated: the tiles appear very differently, but they are still there, and Pixelfed does not appear in the user portal…

**Comment by OniriCorpe** [comment]
the tile is showed for me:
![a screenshot of the Pixelfed tile link in the YunoHost portal](https://github.com/user-attachments/assets/873e499f-62e5-4a14-8d9a-cb8a05790c2e)

or maybe you are talking about "the small 'YunoHost' portal shortcut square on apps"? (which will be removed in YunoHost Bookworm)

**Comment by grossermensch** [comment]
@yalh76 can you please close the issue if this is working now? Thanks
